experimental/packages/api-events/src/types/Event.ts

```diff
@@ -27,6 +28,12 @@ export interface Event {
    */
   name: string;
 
+  /**
+   * Attributes that describe the event.
```

Comment:
Suggest avoiding the term `attributes` here as it's confusing. 
```suggestion
   * The data / payload of the event.
```